source/OctoPack.Tests/OctoPack.Tests.csproj

```diff
@@ -1,10 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\NUnitTestAdapter.2.3.0\build\NUnitTestAdapter.props" Condition="Exists('..\packages\NUnitTestAdapter.2.3.0\build\NUnitTestAdapter.props')" />
```

Comment:
I couldn't run the tests via `build.cmd` or via my IDE with this test adapter.

Reply:
So the Test Adapter is only used when running the tests from within Visual Studio.

Reply:
I amend this comment. The test adapter is used when running the tests using vstest.console.exe (which internally, Visual Studio uses to run the tests, too). So, this implies when performing a build using MSBuild, this test adapter would be used.

Now, when it comes to Cake, I don't know how they implemented their NUnit function. Perhaps it uses the Nunit console?